app/game_manager.py

```diff
@@ -1,3 +1,7 @@
+import ascii_art
```

Comment:
use relative import instead of absolute import as this will not work when using `runney,py ` to run the app 
```suggestion
from . import ascii_art
```

Reply:
yh